### PR TITLE
Add dependabot.yml config

### PR DESCRIPTION
**Pull request by bigint**
Add dependabot.yml config

/cc @bonustrack @samuveth

### CODE REVIEW

**Review by bonustrack — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
```
Comment (samuveth) [inline .github/dependabot.yml:3]: Should this not be yarn?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    commit-message:
+      prefix: '⬆️ deps:'
+      prefix-development: '⬆️ deps(dev):'
+    assignees:
+      - bonustrack
+      - samuveth
+    reviewers:
```
Comment (samuveth) [inline .github/dependabot.yml:13]: Should we add @ChaituVR and @mktcode ?

File: .github/dependabot.yml (hunk 1 above)
Comment (bigint) [reply .github/dependabot.yml:3]: Screenshot 2021-12-17 at 7 59 23 PM

https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#package-ecosystem

npm is for both npm and yarn registries 😄

File: .github/dependabot.yml (hunk 2 above)
Comment (bigint) [reply .github/dependabot.yml:13]: If ok I'll add 😄

Comment (bonustrack) [reply .github/dependabot.yml:13]: 2 reviewer is enough, it should be Sam and Mkt

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    commit-message:
+      prefix: '⬆️ deps:'
+      prefix-development: '⬆️ deps(dev):'
+    assignees:
+      - bonustrack
```
Comment (samuveth) [inline .github/dependabot.yml:11]:
```suggestion
      - mktcode
```

File: .github/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      - samuveth
+    reviewers:
+      - bonustrack
```
Comment (samuveth) [inline .github/dependabot.yml:14]:
```suggestion
      - mktcode
```

**Pull request merged**